modules/pubCommonIdSystem.js

```diff
@@ -223,6 +233,11 @@ export const pubCommonIdSubmodule = {
    * @returns {IdResponse}
    */
   getId: function (config = {}, consentData, storedId) {
+    const coppa = coppaDataHandler.getCoppa();
+    if (coppa) {
+      utils.logInfo('PubCommonId: coppa not provided, exiting PubCommonId');
```

Comment:
should this say "coppa not supported" or "IDs not provided for coppa requests"? to me, this message sounds like "coppa config wasn't provided, so we're existing"